package/CHANGELOG

```diff
@@ -13,13 +13,19 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, BFedder, inomag
+??/??/?? IAlibay, BFedder, inomag, shudipto-amin
 
  * 2.2.0
 
 Fixes
   * Updated the badge in README of MDAnalysisTest to point to Github CI Actions
-
+  * Provides Contacts ability to use static AtomGroups as well as selection 
```

Comment:
Entries should be newest first in the changelog

Reply:
Done, I put it at the top.